docs/conf/scalar-manager-custom-values.yaml

```diff
@@ -0,0 +1,12 @@
+scalarManager:
+  # targets -- The targets that Scalar Manager should monitor
+  targets: # example
+    - name: ledgers
```

Comment:
This is the name that is displayed on the screen, right?
Then, I think we should name it `Ledger`.

Reply:
@feeblefakie 
Yes.
It is used to be displayed in the Scalar Manager.

Reply:
I fixed it in https://github.com/scalar-labs/helm-charts/pull/109/commits/ad6f94a6244462110f72a47a3b89fc4a089a9dfd